### PR TITLE
Adding Travis

### PR DESCRIPTION
**Pull request by NathanDotTo**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  - "2.7"
+  - "3.2"
+  - "3.3"
+  - "3.4"
+  - "3.5"
+  - "3.5-dev" # 3.5 development branch
+  - "3.6"
+  - "3.6-dev" # 3.6 development branch
+  - "3.7-dev" # 3.7 development branch
+  - "nightly" # currently points to 3.7-dev
```
Comment (shacker) [inline .travis.yml]: Does this mean the tests will run repeatedly, once for each listed version of python? If so, do we need that level of thoroughness or could we get away with just 2.7 and 3.6 (for example)?

**Comment by shacker** [comment]
I would vote to squash all the commits in this PR into one, personally (not a huge deal or blocker, just cleaner).

**Comment by NathanDotTo** [comment]
Hi

This will test for each version of Python separately.

We probably could just do 2.7 and 3.6.

I’ll adjust that.

Regards

Nathan

**Comment by shacker** [comment]
Cool. So do you want to squash these, or leave 'em?

**Comment by NathanDotTo** [comment]
Hi, the build passed in my Travis build.

I appreciate that it would be better to squash these, but could we just merge this lot as is please?

Thanks

Nathan

**Comment by shacker** [comment]
@liamks I'm going to let you sign off on / merge this one.

**Comment by shacker** [comment]
Oh - Just realized I can squash and merge from here on github! Sorry for the delay - doing that now.

**Comment by NathanDotTo** [comment]
It seems to be working :-)

**Pull request merged**